mayastor/src/bdev/nexus/nexus_bdev.rs

```diff
@@ -883,11 +886,17 @@ pub async fn nexus_create(
 }
 
 /// Destroy list of child bdevs
-async fn destroy_child_bdevs(name: &str, list: &[String]) {
-    let futures = list.iter().map(String::as_str).map(bdev_destroy);
+async fn destroy_child_bdevs(nexus: &str, list: &[String]) {
+    let futures = list
+        .iter()
+        .map(String::as_str)
+        .filter(|name| Bdev::lookup_by_name(name).is_some())
```

Comment:
Does this properly find our non-bdev NVMx devices ? IIRC, Bdev::lookup_by_name() ends up in 'spdk_bdev_get_by_name()', which can't see our NVMe controllers.

Reply:
Have replaced with device_lookup(), and also used device_destroy().
There no longer seems to be any valid use for bdev_destroy(), and that should probably be removed.